admin-auth/src/test/kotlin/com/epam/drill/admin/auth/TestUtils.kt

```diff
@@ -93,4 +96,14 @@ suspend fun HttpRequestData.formData() = String(this.body.toByteArray())
 fun URL.queryParams() = query?.split("&")?.associate {
     val (key, value) = it.split("=")
     key to value
-} ?: emptyMap()
\ No newline at end of file
+} ?: emptyMap()
+
+fun mockHttpClient(vararg requestHandlers: RequestHandler) = HttpClient(MockEngine { request ->
+    requestHandlers
+        .find { request.url.encodedPath == it.first }
```

Comment:
Using pair's first/second fields always introduces some level of confusion
I suggest to use data class instead, so "first" and "second" would have actual names